### PR TITLE
add/tekrajjoshi

### PR DESCRIPTION
**Pull request by T3KRAJ**

### CODE REVIEW

**Review by anshumanv — CHANGES_REQUESTED** [review]
Can you order it alphabetically?

**Comment by T3KRAJ** [comment]
sure

**Review by anshumanv — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -330,10 +330,12 @@ Hopefully this repo can serve as a source of inspiration for your portfolio!
 ## T
 
 - [Tejas Kumar](http://tej.as)
+- [Tek Raj Joshi](https://t3kraj.netlify.app)
 - [Thea Mushambadze](https://highflyer910.github.io/)
 - [Tom Sherman](https://tom-sherman.com)
 - [Travis Fischer](https://transitivebullsh.it)
 
+
```
Comment (anshumanv) [inline README.md:338]: ah can you remove this extra line just for the sake of consistency?

Comment (T3KRAJ) [reply README.md:338]: ok

**Pull request merged**